### DISCUSSION TITLE
Fatture - Implementazione sconto con importo fisso sulla singola riga di dettaglio

### DISCUSSION THREAD

**Discussion by g-scalvini** [fattureincloud/api#397] [Q&A] [answered]
Buongiorno a tutti,
nella rappresentazione tabellare del tracciato Fattura PA, è indicato che, all'interno degli N elementi della sezione "2.2 DatiBeniServizi" per il blocco 2.2.1.10 ScontoMaggiorazione, è necessario specificare almeno uno tra gli elementi "2.2.1.10.2 Percentuale" o "2.2.1.10.3 Importo", fornendo quindi la possibilità di inserire uno sconto percentuale oppure uno sconto con importo fisso:

> Se valorizzato, viene controllato che esista almeno uno dei due elementi 2.2.1.10.2 Percentuale e 2.2.1.10.3 Importo

All'interno dell'SDK, nella classe IssuedDocumentItemsListItem, il campo "discount" è unico e di fatto corrisponde al campo di sconto percentuale, non permettendo quindi di inserire uno sconto con importo fisso.

L'introduzione dello sconto con importo fisso sulla singola riga di dettaglio è prevista come futura implementazione?

Grazie

**Comment by valmoz** [comment, marked as answer]
Buongiorno @g-scalvini 
purtroppo la nostra piattaforma non supporta questa funzionalità.

Una possibile alternativa può essere quella di aggiungere un nuovo item con cifra negativa, specificando al suo interno che si tratta di uno sconto.

Grazie
Mauro

**Reply by g-scalvini** [reply]
Buongiorno Mauro,
avevamo pensato anche noi all'utilizzo di un nuovo item con importo negativo con la dicitura "Sconto concordato per _nomeProdotto_", che di fatto è similare alla logica che è stata adottata da tutti per gestire gli sconti sul totale fattura.

Ci chiedevamo però se l'inserimento di una riga di sconto per il prodotto venga considerato corretto dall'AdE o chi per lei visto che, rispetto allo sconto globale, le specifiche della Fatturazione Elettronica prevendono già il campo per lo sconto ad importo fisso sulla singola riga.

Grazie
Giorgio

**Reply by emajo** [reply]
Ciao @g-scalvini, sì, l'inserimento di una riga di sconto per il prodotto viene considerato corretto dall'Agenzia delle Entrate, abbiamo numerosi clienti che lo utilizzano in questo modo senza alcun problema.

**Reply by g-scalvini** [reply]
Ciao Emanuele, grazie per il chiarimento.
Procederemo con l'implementazione della soluzione indicata.

Vi ringrazio nuovamente per il supporto.
Buona giornata
Giorgio

**Comment by g-scalvini** [comment]
Chiudo la issue vista la soluzione indicata da @valmoz.
